pkg/ingester/ingester.go

```diff
@@ -62,8 +64,17 @@ type Ingester struct {
 	flushQueuesDone sync.WaitGroup
 }
 
+// ChunkStore is the interface we need to store chunks.
+type ChunkStore interface {
+	Put(ctx context.Context, chunks []chunk.Chunk) error
+}
+
 // New makes a new Ingester.
-func New(cfg Config, store chunk.Store) (*Ingester, error) {
+func New(cfg Config, store ChunkStore) (*Ingester, error) {
+	if cfg.MaxChunkIdle == 0 {
+		cfg.MaxChunkIdle = 30 * time.Minute
+	}
```

Comment:
Is this nessecary?  I prefer to use util.DefaultValues in tests.  And not to repeat the default everywhere...